SofaKernel/modules/SofaHelper/src/sofa/helper/Quater.h

```diff
@@ -101,10 +99,68 @@ class SOFA_HELPER_API Quater
         _q[3]=1.0;
     }
 
-    void fromFrame(defaulttype::Vec<3,Real>& x, defaulttype::Vec<3,Real>&y, defaulttype::Vec<3,Real>&z);
+    void fromFrame(Vector3& x, Vector3&y, Vector3&z);
```

Comment:
I think that we should put the code part in a .inl, have a .cpp and with two explicit extern template.

Reply:
I can do it for some functions (only templated on float/double) but it wont be possible to do it with one other template (typically the matrix type or the vec type)